doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -52,49 +53,102 @@ For this guide, we'll create a getting started project in a branch of your fork
+    clear-output-folder: true
 ```
 
-**Note**:
+#### Create a sdk project folder
+  
+You can manually create the project folder. Please refer to [Azure.Template](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template) as an example.
+
+**Note**: 
+- Please refer to [sdk-directory-layout](https://github.com/Azure/azure-sdk/blob/main/docs/policies/repostructure.md#sdk-directory-layout) for detail information.
+- remove `autorest.md` from sdk\<service name>\<package name>\src
+
+
+#### Generate the library source code
 
-- `-service` takes Azure client service directory name. ie. purview. It equals to the name of the directory in the specification folder of the azure-rest-api-specs repo that contains the REST API definition file.
-- For `- namespace`, please use one of the pre-approved namespace groups on the [.NET Azure SDK Guidelines Approved Namespaces list](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespaces-approved-list). This value will also provide the name for the shipped package, and should be of the form `Azure.<group>.<service>`.
-- `-sdkPath` takes the address of the root directory of sdk repo. e.g. /home/azure-sdk-for-net
-- `-inputfiles` takes the address of the Open API spec files,  separated by semicolon if there is more than one file.  The Open API spec file can be local file, or the web address of the file in the `azure-rest-api-specs` repo.  When pointing to a local file, make sure to use **absolute path**, i.e. /home/swagger/compute.json. When pointing to a file in the `azure-rest-api-specs` repo, make sure to include the commit id in the URI, i.e. `https://github.com/Azure/azure-rest-api-specs/blob/73a0fa453a93bdbe8885f87b9e4e9fef4f0452d0/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json`. This ensures that you can choose the time to upgrade to new swagger file versions.
-- `-readme` takes the address of the readme configuration file. The configuration can be local file, e.g. ./swagger/readme.md or the web address of the file in the `azure-rest-api-specs` repo, i.e. `https://github.com/Azure/azure-rest-api-specs/blob/23dc68e5b20a0e49dd3443a4ab177d9f2fcc4c2b/specification/deviceupdate/data-plane/readme.md`
-- You need to provide one of `-inputfiles` and `-readme` parameters. If you provide both, `-inputfiles` will be ignored.
-- `-securityScope` designates the authentication scope to use if your library supports **Token Credential** authentication.
-- `-securityHeaderName` designates the key to use if your library supports **Azure Key Credential** authentication.
+Generate the library source code files to the directory `<sdkPath>/sdk/<service>/<namespace>/src/Generated`
 
-When you run the `eng\scripts\automation\Invoke-DataPlaneGenerateSDKPackage.ps1` script, it will:
+  Enter `Cadl Project Folder`, run
+  ```
+  npm install
+  npx cadl compile --emit @azure-tools/cadl-csharp --output-path <Path-to-source-code-folder>
```

Comment:
Any command line options we should aware of?

Reply:
I think it is better to ask them to apply additional configuration options in cadl-project.ymal, not in the command line for simplification.